crates/proof-sdk/proof/src/sync.rs

```diff
@@ -26,17 +28,16 @@ where
 
     // Walk back the starting L1 block by `channel_timeout` to ensure that the full channel is
     // captured.
-    let channel_timeout =
-        boot_info.rollup_config.channel_timeout(safe_head_info.block_info.timestamp);
+    let channel_timeout = rollup_config.channel_timeout(safe_head_info.block_info.timestamp);
     let mut l1_origin_number = l1_origin.number.saturating_sub(channel_timeout);
-    if l1_origin_number < boot_info.rollup_config.genesis.l1.number {
-        l1_origin_number = boot_info.rollup_config.genesis.l1.number;
+    if l1_origin_number < rollup_config.genesis.l1.number {
+        l1_origin_number = rollup_config.genesis.l1.number;
     }
     let origin = chain_provider.block_info_by_number(l1_origin_number).await?;
 
     // Construct the cursor.
     let mut cursor = PipelineCursor::new(channel_timeout, origin);
-    let tip = TipCursor::new(safe_head_info, safe_header, boot_info.agreed_l2_output_root);
+    let tip = TipCursor::new(safe_head_info, safe_header, B256::ZERO);
```

Comment:
note to reader - this type is pretty weird. We should refactor this at some point - it's mainly a way to retrieve the OR after the block was computed, but nothing that uses it depends on a starting output root at all.